### PR TITLE
Add Basic Auth to Try Out Tab in DevPortal

### PR DESCRIPTION
**Pull request by kavishkafernando**
Fixes #https://github.com/wso2/product-apim/issues/7580

This PR only adds Basic Auth to the Try Out tab

Screen Shot 2020-02-28 at 11 00 31 AM

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx [hunk 1]
```diff
@@ -481,7 +503,29 @@ class ApiConsole extends React.Component {
...
+                                            />
+                                            <TextField
+                                            margin='normal'
+                                            variant='outlined'
+                                            className={classes.passwordField}
+                                            label={<FormattedMessage id='password' defaultMessage='Password' />}
+                                            name='password'
+                                            onChange={this.handleChanges}
+                                            value={password || ''}
+                                            />
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx]: There is an indentation issue here

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx [hunk 2]
```diff
@@ -518,26 +562,38 @@ class ApiConsole extends React.Component {
...
+                                                      control={<Radio />}
+                                                      label='OAUTH'
+                                                  />
+                                                    )}
+                                                    {isBasicAuthEnabled && (
+                                                        <FormControlLabel
+                                                        value='BASIC'
                                                         control={<Radio />}
-                                                        label='OAUTH'
+                                                        label='BASIC'
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx]: I think capitalizing these labels are wrong, it should be

- Basic
- Referenced ( OAuth )
- API Key
- Mutual SSL

**Comment by ashishpilania18** [comment]
@tmkasun 

  I have used APIM Version wso2am-3.1.0-beta but I am not getting option of Oauth ,Basic , API Key in this version . Please confirm whether it is available in version or not.

Also Used mututal SSL for implementing Security in API ,but its nor working and still showing MutualSSLAuthenticator Mutual SSL authentication failure in logs . Please help 
  
image

**Comment by kavishkafernando** [comment]
Hi @ashishpilania18 

The Basic feature has not yet been merged hence the PR. OAuth and API Key options should be displayed.

Can you please open a git issue regarding the Mutual SSL error. We will be able to provide you further assistance through the git issue.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx (hunk 2 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx]: Done

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx]: Fixed

**Pull request merged**